pipeline-yaml_5b5744d40428631d7a8940a9.md

```diff
@@ -615,6 +614,51 @@ were also defined on the `task` level.
 	          - echo $VAR_1
 	          - echo $APP_ENV
 
+### matrix
+
+The `martix` property allows you to define one or more environment variables
+with multiple values. After that, it creates multiple parallel jobs by
+calculating the Cartesian product of all sets of values.
+
+So, the final outcome of the `matrix` property is parallel jobs with the
+commands that are defined in the respective `jobs` property joined with all
+possible combinations of the environment variables defined in the `matrix`
+property.
```

Comment:
So, the final outcome of the `matrix` property is creation of multiple parallel jobs with the
same commands, defined in the respective `commands` property.
Each job is assigned env vars from corresponding element of Cartesian product.

Reply:
Done.